cpp/src/utilities/cython.cu

```diff
@@ -865,5 +959,24 @@ template void call_sssp(raft::handle_t const& handle,
                         int64_t* predecessors,
                         const int64_t source_vertex);
 
+// TODO: add the remaining relevant EIDIr's:
+//
+template major_minor_weights_t<int32_t, float> call_shuffle(raft::handle_t const& handle,
+                                                            int32_t* edgelist_major_vertices,
+                                                            int32_t* edgelist_minor_vertices,
+                                                            float* edegelist_weights,
```

Comment:
misspelling here (immaterial, but it made me look)

Reply:
Fixed.